site-modules/adhoc/tasks/nix_example.json

```
@@ -0,0 +1,4 @@
+{
+     "description": "This is the description for the nix_example task",
+     "input_method": "environment"
+   }
```

Indentation for JSON: 4 spaces, closing brace at same level as opening brace
```suggestion
    "description": "This is the description for the nix_example task",
    "input_method": "environment"
}
```